### DISCUSSION TITLE
Add alt status bar book info

### DISCUSSION THREAD

**Discussion by codeminatodor** [koreader/koreader#11757] [Q&A]
Pleas tell me how to add book info to alt status bar?
Year from tag <date
<sequence number
and more...

**Comment by Commodore64user** [comment]
I don't think this is currently possible, and I also believe nobody is interested in touching that side of the reading engine. Fuzzy on the specific details just letting you know.

**Reply by Frenzie** [reply]
It's basically a living fossil piece of coolreader legacy.

**Reply by Commodore64user** [reply]
Well there you have it folks. Legacy code nobody wants to deal with. Why not deprecate it?

**Reply by Frenzie** [reply]
Anything that inoffensively hasn't required any maintenance for well over a decade doesn't get on anyone's nerves. Also the other one can't go on top I believe.
